src/app/members/members-data.ts

```diff
@@ -669,4 +669,27 @@ export const members: TMember[] = [
       { name: "Luck", value: Math.floor(Math.random() * 100), max: 100 },
     ],
   },
+  {
+    name: "Shigetora Miyashita",
+    login: "shingy",
+    role: ["Alumni"],
+    imagePath: "/images/members/shingy.png",
+    description: "Graduate Student in Nuclear Engineering and Management at UTokyo",
```

Comment:
Your code is not properly format (on this line). Please try running `npm run lint`. And add a new commit to this branch